.github/workflows/deploy-prod.yml

```diff
@@ -59,3 +66,10 @@ jobs:
           APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
           CLUSTER: prod-fss
           RESOURCE: nais-prod.yaml
+      - name: Post deploy failures to Slack
+        if: failure()
+        run: |
+          curl -X POST --data "{\"text\": \"$MESSAGE - $GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID\"}" $WEBHOOK_URL
+        env:
+          MESSAGE: "Deploy av feilet"
```

Comment:
"Deploy av ? feilet"

Reply:
preprod?

Reply:
Kanskje noe sånt?
`Deploy av familie-ef-sak til prod feilet`

Mulig vi kan bytte ut `familie-ef-sak` og `prod` med variabler. Har iallefall `$GITHUB_REPOSITORY` - vet ikke om `${env.cluster}` kan brukes istedenfor å hardkode miljøet?

Reply:
evt deploy feilet :)

Reply:
Jeg kan leve med en melding som Charlie foreslår over: 
"Deploy av familie-ef-sak til prod feilet"

Reply:
Tenker att vi kun trenger denne for prod, så har hardkodet prod :)